app/templates/components/preprint-form.hbs

```diff
@@ -1,33 +1,33 @@
 {{#with _names.[0] as |name|}}
-    {{#preprint-form-section id='preprint-form-upload' open=true}}
+    {{#preprint-form-section id='preprint-form-upload' open=true name=name}}
         {{preprint-form-header name=name valid=(get valid name)}}
-        {{preprint-form-upload verify=(action 'verify' name)}}
+        {{preprint-form-upload verify=(action 'verify' name) next=(action 'next' name)}}
```

Comment:
What is the advantage of having a single section of the form be a separate component? (do we intend to reuse these form elements in any way?)

Components are nice for isolating reusable elements, but if neither of those concerns apply, they also add complexity (behavior spread across extra files, and an extra layer of reasoning about data flow).